app/src/main/java/com/example/roomremix2/DrawView.kt

```diff
@@ -34,11 +34,17 @@ class DrawView : View {
     private fun init(){
         /* initialize function
         makes texture smooth for drawing */
-        drawTool.isAntiAlias = true
-        drawTool.color = currentColor
-        drawTool.style = Paint.Style.STROKE
-        drawTool.strokeJoin = Paint.Join.ROUND
-        drawTool.strokeWidth = 8f
+
+        /*
+            chained the setters so that it dosnt just say "drawtool." over and over again.
+         */
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.